boards/BMS/Inc/App/App_SetPeriodicCanSignals.h

```diff
@@ -5,12 +5,3 @@
 void App_SetPeriodicCanSignals_Imd(
     struct BmsCanTxInterface *can_tx,
     struct Imd *              imd);
-
-void App_SetPeriodicSignals_AccumulatorInRangeChecks(
-    struct BmsCanTxInterface *can_tx,
-    const struct Accumulator *accumulator,
-    struct ErrorTable *       error_table);
-
-void App_SetPeriodicSignals_CellMonitorsInRangeChecks(
-    struct BmsCanTxInterface * can_tx,
-    const struct CellMonitors *cell_monitors);
```

Comment:
Why was this deleted?

Reply:
Bc we shouldnt use in range checks, i think theyre a little extra

Reply:
I also want to wrap these using functions from App_Accumulator, don't think we need an entirely new file containing functions to broadcast CAN messages